### PR TITLE
Added Issue Templates

### PR DESCRIPTION
**Pull request by whitehacker**
## Purpose of this PR

This pull request introduces standardized issue templates to streamline the issue creation and resolution process in our REMIND repository. The templates cover Feature Requests, Bug Reports and we can add more custom templates if needed. 

These are draft templates and feedback on the template format or content is highly welcomed.

### CODE REVIEW

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — COMMENTED** [review]
Seems like an awful lot of work to make everybody who actually opens REMIND issues click once more often, seeing that the only issues we ever got from outside where about making input data available https://github.com/remindmodel/remind/issues/299, https://github.com/remindmodel/remind/issues/1421.

**Comment by LaviniaBaumstark** [comment]
right, this is a preparation step for starting collaboration with external users to be prepared for their issues and to avoid wrong expectations.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: checkboxes
+    id: terms
+    attributes:
```
Comment (LaviniaBaumstark) [inline .github/ISSUE_TEMPLATE/bug.yml:72]: maybe we should add here that we are happy about any bug report and will try to fix bugs in REMIND soon. But that we con only provide very limited support for setting up and running REMIND outside PIK environment.

Comment (whitehacker) [reply .github/ISSUE_TEMPLATE/bug.yml:74]: Sure, and added.

File: renv/settings.json
Comment (tonnrueter) [inline renv/settings.json:1]: I do not understand how the `renv` settings file is relevant to this PR. I suspect that the presence of this is due to an accidental commit

**Pull request closed without merging**